.github/workflows/npm-publish.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           NODE_ENV: production
       - name: The final publish step within dist folder
         working-directory: dist
-      # This will publish the package and set access to public as if you had run npm access public after publishing.
```

Comment:
Please also add the npm run test as part of the publish workflow

Reply:
will do